packages/icons/src/index.js

```diff
@@ -305,303 +309,3302 @@ import { sizes, AVAILABLE_SIZES } from "./sizes";
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <ZoomOut24SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const AddFolder16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <AddFolder16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const AddMember16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <AddMember16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Add16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <Add16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Archive16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <Archive16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const AreaPost16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <AreaPost16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Assets16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <Assets16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Attach16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <Attach16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Attachment16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <Attachment16SVG {...propsClone} />;
+    }}
+  </ThemeContext.Consumer>
+);
+
+export const Back16 = props => (
+  <ThemeContext.Consumer>
+    {({ resolvedRoles }) => {
+      const propsClone = Object.assign({}, props, {
+        className: cx(css(stylesheet(resolvedRoles)), props.className)
+      });
+      return <
```

Comment:
It might actually be better for tree-shaking to splay these all out like this. But `src/index.js` is the file that `writeBundle.js` generates so there's a limit to how clever this code can be.